mcs/class/corlib/System/Enum.cs

```diff
@@ -392,7 +392,7 @@ public static string[] GetNames (Type enumType)
 			MonoEnumInfo.GetInfo (enumType, out info);
 			return (string []) info.names.Clone ();
 		}
-
+/*
```

Comment:
Yes, sorry, this could use some explanation. The first implementation of this method was the one I originally changed to work around the problem. That change caused a consistent failure in one test that I could not explain. So I have reverted that change so that this function will again cause problems for enums with IL2CPP. I then commented out the problematic method, and used the second implementation of the method, with a change that corrects the problem.